lessons/07-tests-tasks/inclass/tests/server/appSpec.js

```diff
@@ -0,0 +1,35 @@
+var request = require('supertest');
+var app = require('./../../app.js');
+
+describe("The app", function() {
+  it('should return 200 OK on GET /', function(done) {
+    request(app)
+      .get('/')
+      .expect(200)
+      .expect('Content-Type', 'text/html; charset=utf-8')
+      .expect('Content-Length', '515')
```

Comment:
This test is asking for a 200 response, checking content-length (or better yet, the actual content) is more specific and something I would break into a separate test.